index.html

```diff
@@ -146,25 +166,54 @@ <h3>LET’S CONNECT</h3>
           <li><a href="#"><i class="fa fa-github-alt fa-lg"></i></a></li>
           <li><a href="#"><i class="fa fa-angellist fa-lg"></i></a></li>
         </ul>
-        <button>Get my resume</button>
-        <div>
-          <ul class="skills">
-            <li> <p class="list"><span>Language</span>  <i class="fa-solid fa-chevron-down"></i></p> 
-              <ul class="language">
-                <li><a href="#"><img src="assets/images/js.svg" alt=""> javascript</a></li>
-                <li><a href="#"> <img src="assets/images/html.svg" alt=""> html</a></li>
-                <li><a href="#"><img src="assets/images/css.svg" alt=""> css</a></li>
-              </ul>
-            </li>
-            <li><p class="list"><span>Frameworks</span>  <i class="fa-solid fa-chevron-right"></i></p></li>
-            <hr>
-            <li><p class="list"><span>Skills</span>  <i class="fa-solid fa-chevron-right"></i></p></li>
-          </ul>
-          <hr>
-        </div>
+      </div>
+                  <button><h3>Get my resume</h3></button>
+      <div>
+        <ul class="skills">
+          <li> <p class="chev"><span>Language</span>  <i class="fa-solid fa-chevron-down"></i></p> 
+            <ul class="language">
+              <li><a href="#"><img src="assets/images/js.svg" alt=""> javascript</a></li>
+              <li><a href="#"> <img src="assets/images/html.svg" alt=""> html</a></li>
+              <li><a href="#"><img src="assets/images/css.svg" alt=""> css</a></li>
+            </ul>
+          </li>
+          <li><p class="chev"><span>Frameworks</span>  <i class="fa-solid fa-chevron-right"></i></p> <hr></li>
+          <li><p class="chev"><span>skills</span>  <i class="fa-solid fa-chevron-right"></i></p> <hr>
+          </li>
+        </ul>
       </div>
       </div>
     </div>
   </section>
-</body>
+  <footer class="contact-me" id="contact">
+    <section class="contact">
+      <div class="container">
+
+        <!-- contact text -->
+        <div class="contact-text">
+          <h2>Contact Me</h2>
+          <p>If you have an application you are interested in developing, a feature that you need built or a project
+            that needs coding. I’d love to help with it!</p>
+        </div>
+
+          <!-- contact form -->
+        <div class="contact-form">
+          <form action="https://formspree.io/f/xbjewoav" method="post">
+            <div class="form-field">
+              <input type="text" name="name" placeholder="Full Name" maxlength="30" required>
+            </div>
+            <div class="form-field">
+              <input type="email" name="email" placeholder="Email" required>
+            </div>
+            <div class="form-field">
+              <textarea name="message" placeholder="Write your message here..." maxlength="500" required></textarea>
+            </div>
+            <div class="form-field">
+              <button type="submit">Get in touch </button>
```

Comment:
- [OPTIONAL] I am going to make this an optional change because it wasn't pointed out by the previous reviewer but I'll suggest you implement this before merging this pull request. 
- Currently, your button doesn't have the interactive state stylings added to it `Hover, pressed`, I'll suggest you implement this as it's part of the requirement and also gives a good user experience. :smile: :smile: 

- Here is a screenshot of the states from the Figma design:
![Screenshot from 2023-05-11 23-55-57](https://github.com/Izebiafe/my-protfolio/assets/107632780/f3489fcc-bd20-4d5e-8a11-ce50f7e3b3ac)